README.md

```diff
@@ -73,39 +78,27 @@ pip install -r requirements/gui.txt
 After this, the command `python joinmarket-qt.py` from within the `scripts` subdirectory should work.
 There is a [walkthrough](docs/JOINMARKET-QT-GUIDE.md) for what to do next.
 
-### Notes on architectural changes (can be ignored)
+### Architecture notes
 
-Motivation: By separating the code which manages conversation with other
-Joinmarket participants from the code which manages this participant's Bitcoin
-wallet actions, we get a considerable gain at a minor cost of an additional layer:
-code dependencies for each part are much reduced, security requirements of the 
-server/daemon layer are massively reduced (which can have several advantages such as
-it being more acceptable to distribute this layer as a binary), and client code
-can be written, implementing application-level logic (do join with coins X under condition X)
-using other Bitcoin libraries, or wallets, without knowing anything about
-Joinmarket's inter-participant protocol. An example is my work on the Joinmarket
-electrum [plugin](https://github.com/AdamISZ/electrum-joinmarket-plugin).
+See [architecture-notes.md](docs/architecture-notes.md).
 
-It also
-means that updates to the Bitcoin element of Joinmarket, such as P2SH and segwit, should
-have extremely minimal to no impact on the backend code, since the latter just implements
-communication of a set of formatted messages, and allows the client to decide on
-their validity beyond simply syntax.
+### TESTING
 
-Joinmarket's own [messaging protocol](https://github.com/JoinMarket-Org/JoinMarket-Docs/blob/master/Joinmarket-messaging-protocol.md) is thus enforced *only* in the server/daemon.
+Instructions for developers for testing [here](docs/TESTING.md). If you want to help improve the project, please have a read of [this todo list](docs/TODO.md).
```

Comment:
Maybe also add link to https://github.com/JoinMarket-Org/joinmarket-clientserver/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22 in addition to docs/TODO.md?